cli/lsp/documents.rs

```diff
@@ -911,35 +929,32 @@ impl DocumentsInner {
     specifiers: Vec<String>,
     referrer: &ModuleSpecifier,
   ) -> Option<Vec<Option<(ModuleSpecifier, MediaType)>>> {
-    let doc = self.get(referrer)?;
+    let dependencies = self.get(referrer)?.0.dependencies.clone();
```

Comment:
Nitpick: Can `doc` still be stored then `doc.0.dependencies` be used below instead to not clone? I think it should be ok because `Document` is immutable in an `Arc` (so should also be no borrow checker trouble). I see previously it was cloning and I think it's just something I missed in my refactor putting `Document` (`DocumentInner`) into an `Arc`.

Reply:
No, because it is a double borrow of self.

Reply:
Oh yeah, I forgot the return type is `&Document`. Could do `let doc = self.get(referrer)?.clone()` which is very cheap then.

Anyway, not a big deal.

Reply:
Dependencies in the document is an Arc now too.